src/data/docs/commands.md

```diff
@@ -44,3 +44,80 @@ Here's what these reasons mean:
 #### I want to `undo` a partially completed `add` where some of the definitions failed
 
 Just do [`undo`](#undo)!
+
+## `docs`
+
+Use `docs foo` to print the documentation [associated with](#link) definition `foo` (see [documentation literals](language-reference#documentation-literals) to learn more).
+
+## `find` (and relatives)
+
+The [`find`](#find) command and its relatives ([`find.patch`](#find-patch) and [`find.verbose`](#find-verbose)) can be used to explore the definitions in a Unison codebase.
+
+### `find`
+
+With no arguments, the `find` command will list all definitions in the current namespace (including definitions in descendants of the current namespace).
+
+With one or more arguments (e.g., `find foo bar baz`), the command will list definitions in the current namespace with names similar to the arguments:
+
+```unison
+isEmpty : [a] -> Boolean
+isEmpty l = case l of 
+  [] -> true
+  _ -> false
+```
+
+```ucm
+.> find emp
```

Comment:
Will these be automatically run through the transcript runner when the site is generated?

Reply:
We don't currently have something like that set up, but I think it is a goal.